src/frontend/lib/views/myfields/fields_list.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
```

Comment:
Is Fields List a component for all fields that exist, and MyFields for the fields of the user?

Reply:
No, FieldList is an inner widget of the MyFields widget, listing all fields of the user